hackers.md

```diff
@@ -3,3 +3,4 @@
 | anshumaankrprasad76@gmail.com | Anshumaan Kumar Prasad | [@aminostwt](https://twitter.com/aminostwt) | [@amino19](https://replit.com/@amino19) | Just Right. Thanks! :) |
 | priyankaafssulur@gmail.com | Priyanka Prasad | [@Priyanka_p10](https://twitter.com/Priyanka_p10) | [@Priyankaprasad](https://replit.com/@Priyankaprasad) | Super Excited for this! :) |
 | akshatnema.school@gmail.com | Akshat Nema | [@AksNema](https://twitter.com/AksNema) | [@AKSHATNEMA](https://replit.com/@AKSHATNEMA) | My hands are desperate to code |
+| prashant.th02@gmail.com | Prashant Thakur | [@Prashant_th18](https://twitter.com/Prashant_th18) | [@PrashantThakur4](https://replit.com/@PrashantThakur4) | |
```

Comment:
Looks like the quote is missing here. Isn't it obligatory? 🤔

Reply:
Should I add a quote @amino19 ??